tests/test_model_helpers/activations/test___init__.py

```diff
@@ -292,6 +292,25 @@ def test_from_stimulus_set(model_ctr, layers, pca_components):
         assert len(activations['neuroid']) == pca_components * len(layers)
 
 
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("number_of_trials", [3, 10])
+@pytest.mark.parametrize(["model_ctr", "layers"], models_layers)
+def test_microsaccades_from_stimulus_set(model_ctr, layers, number_of_trials):
```

Comment:
Previously, microsaccade tests covered only `from_paths`, while the prototypical way to use `ActivationsExtractorHelper` is through `StimulusSets`. Together with the old test, microsaccades are now tested through both the flows; `from_paths` and `from_stimulus_set`.